src/Compilers/Core/Portable/Symbols/IPropertySymbol.cs

```diff
@@ -55,6 +55,11 @@ public interface IPropertySymbol : ISymbol
         /// </summary>
         ITypeSymbol Type { get; }
 
+        /// <summary>
+        /// Gets the declared nullability of the property
+        /// </summary>
+        Nullability DeclaredNullability { get; }
```

Comment:
i feel like this shuld be TypeNullability.  I don't know what 'Declared' means here.